app/models/rbac.rb

```diff
@@ -454,6 +454,8 @@ def self.search(options = {})
   def self.method_with_scope(ar_scope, options)
     if ar_scope == VmdbDatabaseConnection
```

Comment:
I think this can just be `ar_scope == VmdbDatabaseConnection || ar_scope == VmdbDatabaseSetting`